app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -195,23 +271,33 @@ class SupplementalApplicationPage extends React.Component {
 
     const context = {
       application: persistedApplication,
+      applicationMembers: getApplicationMembers(persistedApplication),
+      amis: amis,
+      amiCharts: amiCharts,
+      availableUnits: availableUnits,
       statusHistory: statusHistory,
+      fileBaseUrl: fileBaseUrl,
+      loading: loading,
+      setLoading: this.setLoading,
       onSubmit: this.handleSaveApplication,
       onSavePreference: this.handleSavePreference,
       confirmedPreferencesFailed: confirmedPreferencesFailed,
       onDismissError: this.handleDismissError,
-      fileBaseUrl: fileBaseUrl,
-      amiCharts: amiCharts,
-      amis: amis,
-      availableUnits: availableUnits,
-      loading: loading,
+      statusModal: statusModal,
       openAddStatusCommentModal: this.openAddStatusCommentModal,
       openUpdateStatusModal: this.openUpdateStatusModal,
-      setLoading: this.setLoading,
-      statusModal: statusModal,
       handleStatusModalClose: this.handleStatusModalClose,
       handleStatusModalStatusChange: this.handleStatusModalStatusChange,
-      handleStatusModalSubmit: this.handleStatusModalSubmit
+      handleStatusModalSubmit: this.handleStatusModalSubmit,
+      rentalAssistances: rentalAssistances,
+      addNewRentalAssistance: addNewRentalAssistance,
+      showAddRentalAssistanceBtn: showAddRentalAssistanceBtn,
+      hideAddRentalAssistanceBtn: this.hideAddRentalAssistanceBtn,
+      handleOpenRentalAssistancePanel: this.handleOpenRentalAssistancePanel,
+      handleCloseRentalAssistancePanel: this.handleCloseRentalAssistancePanel,
+      handleSaveNewRentalAssistance: this.handleSaveNewRentalAssistance,
+      handleUpdateRentalAssistance: this.handleUpdateRentalAssistance,
+      handleDeleteRentalAssistance: this.handleDeleteRentalAssistance
```

Comment:
good question! perhaps it's time to revisit the larger question of state management. is it maybe time to graduate to a next level of state management? or maybe we can use Context in a more organized way? this would be a good post-stand-up discussion to have soon